Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM golang:1.18-alpine
+
+WORKDIR /app
+
+COPY go.mod go.sum ./
+
+RUN go mod download
+
+COPY *.go ./
+COPY tscanner ./tscanner
+
+RUN CGO_ENABLED=0 GOOS=linux go build -o Terrapin-Scanner
+
+ENTRYPOINT [ "/app/Terrapin-Scanner" ]
```

Comment:
this can be further optimized by using a multi stage dockerfile
copying the binary (since it is statically linked) to the scratch (or some chainguard or distroless image) drastically reducing the size of the

Reply:
Doing a couple of quick tests this can run fine from a scratch image.

https://github.com/sollie/Terrapin-Scanner/blob/dockertweaks/Dockerfile